src/lib/file.js

```diff
@@ -143,11 +166,32 @@ function addOriginalRepresentation(file) {
 }
 
 /**
- * Wrapper for caching a file object.
+ * Helper to get cache key based on file ID or file version ID. Pass in one or the other.
+ *
+ * @private
+ * @param {Object} options - Cache key options
+ * @param {string} fileId - Get cache key by file ID
+ * @param {string} fileVersionId - Get cache key by file version ID
+ * @return {string} Cache key to use
+ */
+function getCacheKey({ fileId, fileVersionId }) {
+    if (fileId) {
+        return `file_${fileId}`;
```

Comment:
Maybe this function can be reused for tokens.js
Probably export it here or in util?